tests/trainer/test_trainer.py

```diff
@@ -1855,6 +1855,7 @@ def test_torchdynamo_full_eval(self):
         self.assertAlmostEqual(metrics["eval_loss"], original_eval_loss)
         torchdynamo.reset()
 
+    @unittest.skip("torch 2.0.0 gives `ModuleNotFoundError: No module named 'torchdynamo'`.")
```

Comment:
Does this mean `require_torchdynamo` doesn't work for torch 2.0.0?

Reply:
But in this PR, I would just focus on make what needs to be done for CI to run with torch 2. We can do housekeeping later :-)